packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Arrange from 'terra-arrange';
+import ResponsiveElement from 'terra-responsive-element';
+import NavHelpContent from './NavHelpContent';
+import IconInfo from '../../icons/IconInfo';
+import Modal from '../modal/Modal';
+import Popup from '../popup/Popup';
```

Comment:
This is confusing that we have both 
`import Popup from '../popup/Popup';` in this component, but also 
`import Popup from 'terra-popup';` https://github.com/cerner/terra-consumer/pull/6/files#diff-328c90881807f447a8ea18e6f23ab9f1R7
I would think we would want to distinguish between them. If we have our own custom popup, lets name it as such